compiling latest source failed with public_key.hrl not found error

**Describe the bug**
compiling latest source failed with the following error:

=== Entering application ssl
make[3]: *** No rule to make target `/loca../../public_key/include/public_key.hrl', needed by `../ebin/ssl_crl_cache_api.beam'.  Stop.

**To Reproduce**
1. clone the latest source code to a linux machine
2. follow the instruction to build 
3. make will cause the above error 

**Expected behavior**
make successfully build

**Affected versions**
latest source tree

**Additional context**
Add any other context about the problem here. If you wish to attach Erlang code you can either write it directly in the post using [code tags](https://guides.github.com/features/mastering-markdown/), create a [gist](https://gist.github.com), or attach it as a zip file to this post.

--- a/lib/ssh/src/Makefile
+++ b/lib/ssh/src/Makefile
@@ -152,7 +152,7 @@ $(BEHAVIOUR_TARGET_FILES_2): $(BEHAVIOUR_TARGET_FILES_1)
 $(DEP_FILE): $(ERL_FILES)
 	@echo SED $(TARGET) $(ERL_TOP_NATIVE)
 	$(gen_verbose)erlc -M $(ERL_FILES) \
-	| sed "s@[a-zA-Z]\?$(ERL_TOP_NATIVE)@../../..@g" \
+	| perl -pe "s@ [a-zA-Z]?$(ERL_TOP_NATIVE)/(?:bootstrap/)?lib/([^/]+)@ ../../\1@g" 2> /dev/null \
 	| sed "s/\.$(EMULATOR)/\.$$\(EMULATOR\)/" \
 	| sed 's@^ssh_@$$(EBIN)/ssh_@' \
 	| sed 's@^sshc_@$$(EBIN)/sshc_@' \

--- a/lib/ssl/src/Makefile
+++ b/lib/ssl/src/Makefile
@@ -182,7 +182,7 @@ $(TARGET_FILES): $(BEHAVIOUR_TARGET_FILES)
 $(DEP_FILE): $(ERL_FILES)
 	@echo SED $(TARGET) $(ERL_TOP_NATIVE)
 	$(gen_verbose)erlc -M $(ERL_FILES) \
-	| sed "s@[a-zA-Z]\?$(ERL_TOP_NATIVE)/\(bootstrap/\)\?lib/\([^/]\+/\)@../../\2@g" \
+	| perl -pe "s@ [a-zA-Z]?$(ERL_TOP_NATIVE)/(?:bootstrap/)?lib/([^/]+)@ ../../\1@g" 2> /dev/null \
 	| sed "s/\.$(EMULATOR)/\.$$\(EMULATOR\)/" \
 	| sed 's@^dtls_@$$(EBIN)/dtls_@' \
 	| sed 's@^inet_@$$(EBIN)/inet_@' \

